server/games/game.py

```diff
@@ -1,9 +1,21 @@
+from __future__ import annotations
```

Comment:
Oh does this add forward references? That's pretty nice, although I'm hoping to migrate to python 3.9 soon.

Reply:
I believe this will become the default behavior only in python 3.10

Reply:
Or even 3.11: [PEP 563: Correct references to 3.10 default adoption](https://github.com/python/peps/pull/2086)